src/jit/importer.cpp

```diff
@@ -11226,6 +11226,23 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 #endif // _TARGET_64BIT_
                 }
 
+#ifdef FEATURE_CORECLR
+                if ((oper == GT_ADD) || (oper == GT_SUB) || (oper == GT_MUL) || (oper == GT_DIV) || (oper == GT_MOD))
```

Comment:
In that case, it may be worth creating a static version of `GenTree::OperIsArithmetic` (which includes udiv/umod and the logical ops) and using that for the check.